docs/_docs/grpcapiconfiguration.md

```diff
@@ -66,7 +66,10 @@ The following is equivalent to the basic [usage example](usage.html) but without
 3. Generate gRPC stub as before
 
     ```sh
-    protoc -I. --go_out=plugins=grpc,paths=source_relative:./gen/go/ your/service/v1/your_service.proto
+    protoc -I . \
+      --go_out=paths=source_relative:./gen/go/ \
+      --go-grpc_out=paths=source_relative:./gen/go/ \
+      your/service/v1/your_service.proto
```

Comment:
I know you didn't write this, but do you think we could rewrite this in the `--x-opt` format?
```suggestion
    protoc -I . \
      --go_out ./gen/go/ \
      --go_opt paths=source_relative \
      --go-grpc_out ./gen/go/ \
      --go-grpc_opt paths=source_relative
      your/service/v1/your_service.proto
```
I'd like to use this consistently across our documentation because I think it's much clearer. What do you think?

Reply:
I also prefer the `--plugin-opt` format. I did not want to change the instructions, but if you also prefer it, I'll do it!